Directory.Build.props

```diff
@@ -6,8 +6,8 @@
     <NewtonsoftJsonVersion>13.0.1</NewtonsoftJsonVersion>
   </PropertyGroup>
   <PropertyGroup>
-    <AwsSdkSnsVersion>3.7.0</AwsSdkSnsVersion>
-    <AwsSdkSqsVersion>3.7.0</AwsSdkSqsVersion>
+    <AwsSdkSnsVersion>3.7.100</AwsSdkSnsVersion>
+    <AwsSdkSqsVersion>3.7.100</AwsSdkSqsVersion>
```

Comment:
As noted in the previous PR, will this work with just a minimum of `3.7.3`?

Reply:
For SQS, there is no 3.7.3 😢 

Is it OK to have SQS & SNS in different versions?

```c#
    <AwsSdkSnsVersion>3.7.3</AwsSdkSnsVersion>
    <AwsSdkSqsVersion>3.7.0</AwsSdkSqsVersion>
```

Reply:
SQS [3.7.1.43](https://www.nuget.org/packages/AWSSDK.SQS/3.7.1.43) came out on the same day as [3.7.3](https://www.nuget.org/packages/AWSSDK.SimpleNotificationService/3.7.3) for SNS.

SQS [3.7.2](https://www.nuget.org/packages/AWSSDK.SQS/3.7.2) then came out a week later.

We should use whichever the earliest version that has the batch functionality is. You might have to experiment to find which one it is.